src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -32,25 +34,60 @@ public PublicationService(ArtefactRepository artefactRepository,
 
     /**
      * Method that handles the creation or updating of a new publication.
+     *
      * @param artefact The artifact that needs to be created.
-     * @param payload The paylaod for the artefact that needs to be created.
+     * @param payload  The paylaod for the artefact that needs to be created.
```

Comment:
Not part of this PR :) But while you're in this file, could you update the typo in payload

Reply:
Sorted :)